docs/cli.md

```diff
@@ -456,10 +457,10 @@ gcli group delete --group new_group
 gcli tag details --tag tagA
 ```
 
-#### Create a tag
+#### Create tags
 
 ```bash
- gcli tag create --tag tagA
+ gcli tag create --tag tagA tagB
```

Comment:
I don't like this syntax ...
It could have been `gcli tags create tagA tagB`, i.e. `--tag` is not necessary.

Reply:
Add `--tag` for easy identification under multiple parameters.